### PR TITLE
[EC-IIS-4.0.0] Update to edited docs

### PR DESCRIPTION
**Pull request by jsmith-cb**
The docs have been edited at some point in the generated files. This PR back ports those changes to the source content. 

This content has been updated to match https://github.com/cloudbees/docsite-cloudbees-common-sda/pull/615.

### CODE REVIEW

File: help/help.adoc [hunk 1]
```diff
@@ -183,33 +183,33 @@ Creates a new virtual directory in the specified website or updates the existed
...
 
-image::cloudbees-common::cd-plugins/ec-iis/createvirtualdirectory/outcome.png[image]
+image::cloudbees-common-sda::cd-plugins/ec-iis/createvirtualdirectory/outcome.png[role="screenshot"]
 
 Logs will be similar to the following:
 
 ....
 Using plugin EC-IIS-3.1.0.0
-Got parameter "appname" with value "Default Web Site/"
+Got parameter "appname" with value "Default Website/"
```
Comment (alexey-sinyuk) [inline help/help.adoc:196]: these are logs examples
no need to fix mistakes

File: help/help.adoc [hunk 2]
```diff
@@ -183,33 +183,33 @@ Creates a new virtual directory in the specified website or updates the existed
...
 Got parameter "createDirectory" with value "1"
 Got parameter "credential" with value "credential"
 Got parameter "path" with value "mydir"
-Got parameter "physicalpath" with value "c:/tmp/path"
+Got parameter "physical-path" with value "c:/tmp/path"
```
Comment (alexey-sinyuk) [inline help/help.adoc:200]: it's not name of parameter as it appears
these are logs, no need to fix mistakes

File: help/help.adoc [hunk 3]
```diff
@@ -183,33 +183,33 @@ Creates a new virtual directory in the specified website or updates the existed
...
 Going to create directory "C:\tmp\path"
 Directory "C:\tmp\path" already exists, skipping
-Virtual directory Default Web Site/mydir does not exists, proceeding to creating it
+Virtual directory Default Website/mydir does not exist, proceeding to creating it
```
Comment (alexey-sinyuk) [inline help/help.adoc:203]: same

File: help/help.adoc [hunk 4]
```diff
@@ -183,33 +183,33 @@ Creates a new virtual directory in the specified website or updates the existed
...
 Wrote command to property cmdLine
-Going to run command: "C:\WINDOWS\system32\inetsrv\appcmd" add vdir /app.name:"Default Web Site/" /path:"/mydir" /physicalPath:"C:\tmp\path"
+Going to run command: "C:\WINDOWS\system32\inetsrv\appcmd" add vdir /app.name:"Default Website/" /path:"/mydir" /physicalPath:"C:\tmp\path"
```
Comment (alexey-sinyuk) [inline help/help.adoc:205]: same

File: help/help.adoc [hunk 5]
```diff
@@ -183,33 +183,33 @@ Creates a new virtual directory in the specified website or updates the existed
...
 Exit code: 0
-STDOUT: VDIR object "Default Web Site/mydir" added
+STDOUT: VDIR object "Default Website/mydir" added
```
Comment (alexey-sinyuk) [inline help/help.adoc:207]: same
"Default Web Site" is the name of IIS default website

File: help/help.adoc [hunk 6]
```diff
@@ -218,10 +218,10 @@ STDERR: N/A
 
 Creates or updates and starts an in-process web application in the given directory. This procedure assumes that the specified application path exists as a virtual directory.
 
-[cols=",",options="header",]
+[cols=",",options="header"]
 |===
 |Parameter |Description
-|Website Name |The name of the website to add the application, i.e: 'Default Web Site'.
+|Website Name |The name of the website to add the application, i.e: 'Default Website'.
```
Comment (alexey-sinyuk) [inline help/help.adoc:224]: Default Web Site

**Review by alexey-sinyuk — CHANGES_REQUESTED** [review]
Rollback all changes for logs examples.
Change to "Default Web Site" where it was changed 
image

File: help/help.adoc (hunk 1 above)
Comment (jsmith-cb) [reply help/help.adoc]: Updated.

File: help/help.adoc (hunk 2 above)
Comment (jsmith-cb) [reply help/help.adoc]: Updated.

File: help/help.adoc (hunk 3 above)
Comment (jsmith-cb) [reply help/help.adoc]: Updated.

File: help/help.adoc (hunk 4 above)
Comment (jsmith-cb) [reply help/help.adoc]: Updated.

File: help/help.adoc (hunk 5 above)
Comment (jsmith-cb) [reply help/help.adoc]: Updated.

File: help/help.adoc (hunk 6 above)
Comment (jsmith-cb) [reply help/help.adoc]: Updated.

**Review by alexey-sinyuk — APPROVED** [review]

**Pull request merged**